src/main/java/com/google/sps/data/User.java

```diff
@@ -22,11 +29,35 @@ public boolean equals(User user) {
   }
 
   public void addProductArea(String toAdd) {
-      productArea.add(toAdd);
+      specialties.add(toAdd);
   }
 
-  public Collection<String> getProductAreas() {
-      return productArea;
+  public Collection<String> getProductArea() {
+      return specialties;
+  }
+
+  public void setID(long id) {
+      this.id = id;
+  }
+ 
+  public void setProfileType(String profileType) { 
+      this.profileType = profileType;
+  }
+ 
+  public void setSchool(String school) { 
+      this.school = school;
+  }
+ 
+   public void setMajor(String major) { 
+      this.major = major;
+  }
+ 
+  public void setCompany(String company) {
+      this.company = company;
+  }
+ 
+  public void setSpeciality(String specialty) {
+      this.specialty = specialty;
   }
```

Comment:
I think it would be good to add get methods for all the variables so that it is easy to get information when necessary.

Reply:
Done.